docker/proxy/Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM golang:1.19 AS builder
+FROM --platform=${BUILDPLATFORM} golang:1.19 AS builder
```

Comment:
Hi @Zoey2936,
I happened to test the docker(-compose) version atm and realized that the initial build did fail for 2 reasons. One of which is the use of args here and the implication that buildkit is being used. It's not mentioned in the docs though (as far as I have seen).
I'm not sure if this commit should've included a note in the [README.md](https://github.com/strukturag/nextcloud-spreed-signaling#running-with-docker) in the Docker section.

Reply:
that running the compose file requires builds or what exactly?

Reply:
I just added #408 to fix CI and mention that docker-compose v2 is required for building.

See https://github.com/strukturag/nextcloud-spreed-signaling/commit/2582e4ffb44de622b66f21007ee784161c9c7dd1 for discussion.

Reply:
@swixfort did the build succeed with a newer version of docker-compose?

Reply:
I need to check on a different machine because I did manage to alter the config for my setup earlier.

Reply:
@fancycode I did successfully compose with v2.

@Zoey2936 basically, though with docker-compose v2 and the addition from @fancycode it's solved now :-)

Reply:
Great, thanks for confirming.